src/blocks/author-list/components/AnimationControls.js

```
@@ -0,0 +1,87 @@
+				label={ __( 'Hover Effect', 'author-profile-blocks' ) }
+				value={ hoverEffect }
+				options={ hoverEffects }
+				onChange={ ( value ) => onChange( { hoverEffect: value } ) }
+				help={ __(
+					'Visual effect when hovering over list items',
+					'author-profile-blocks',
+				) }
+			/>
+		</PanelBody>
+	);
+}
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)